backend/api/admin/src/main/java/co/airy/core/api/admin/sources/twilio/TwilioSmsSource.java

```diff
@@ -0,0 +1,33 @@
+package co.airy.core.api.admin.sources.twilio;
+
+import co.airy.core.api.admin.Source;
+import co.airy.core.api.admin.SourceApiException;
+import co.airy.core.api.admin.dto.ChannelMetadata;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class TwilioSmsSource implements Source {
+    @Override
+    public String getIdentifier() {
+        return "sms.twilio";
```

Comment:
I'm wondering if `twilio.sms` would work better (no strong opinion... just honest doubt)

Reply:
I have been wondering the same thing. It seems more natural since the "channel provider" is the namespace.

Reply:
fair!